docker-compose.yml

```diff
@@ -17,6 +17,8 @@
 # limitations under the License.
 
 version: '3'
+volumes:
+  keycloak-data: {}
```

Comment:
this just makes the volume to be mounted as root and the file write fails again:

```
[jboss@57bab8d1bca9 ~]$ ls -la /data
total 72
drwxr-xr-x 2 root  root   4096 Sep 18 19:17 .
drwxr-xr-x 1 root  root   4096 Sep 18 19:17 ..
-rwxr-xr-x 1 jboss jboss  1168 Sep 18 14:53 docker-entrypoint.sh
-rw-r--r-- 1 jboss jboss 59979 Sep 18 08:46 renga-realm.json.tpl
```